tensorflow/lite/toco/tflite/export.cc

```diff
@@ -222,7 +222,7 @@ Offset<Vector<Offset<Tensor>>> ExportTensors(
 
     std::vector<int> shape;
     if (array.has_shape()) {
-      for (int d : array.shape().dims()) {
+      for (const auto& d : array.shape().dims()) {
```

Comment:
While you are it could you please just reserve shape right before the for loop, since we know it will be pushing in array.shape().dims().size() elements.